src/Symfony/Component/Debug/Tests/ErrorHandlerTest.php

```diff
@@ -336,6 +336,11 @@ public function testHandleException()
         }
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Support for stacking errors is deprecated since Symfony 3.4 and will be removed in 4.0.
+     * @expectedDeprecation Support for unstacking errors is deprecated since Symfony 3.4 and will be removed in 4.0.
```

Comment:
test are failing, partly because this isn't triggered anymore since the stacking calls are removed from DebugClassLoader
I think the entire test case should be remove
dunno about the other failure yet